cmd/kpod/attach.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	attachFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "detach-keys",
+			Usage: "Override the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [, , or _.",
+		},
+		cli.BoolFlag{
+			Name:  "--no-stdin",
+			Usage: "Do not attach STDIN. The default is false.",
+		},
+	}
+	attachDescription = "Attach to a running container"
```

Comment:
How about 
`The kpod attach command allows you to attach to a running container using the container's ID or name, either to view its ongoing output or to control it interactively. `

Reply:
fixed